docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,248 @@ The following table shows how risk levels are applied to a host, based on the no
+[role="screenshot"]
+image::images/hosts-by-risk-details-page.png[Host risk score on the Hosts by risk tab]
 
-The *Overview* tab on the Alert details flyout:
+You can also visualize host risk score data using prebuilt dashboards that are automatically imported once the feature is enabled. 
+
+To access the dashboards:
+
+. In {kib}, go to *Analytics* -> *Dashboard*, then search for `risk score`.
+. Select *Drilldown of Host Risk Score* to drill down further into risk components of a particular host, or *Current Risk Score for Hosts* to display a list of current risky hosts in your environment.
 
 [role="screenshot"]
-image::images/score-in-flyout.png[]
+image::images/select-hrs-dashboard.png[Select host risk score dashboard]
 
-The *Host risk classification* column in the All hosts table on the Hosts page:
+In this example, we'll explore the *Drilldown of Host Risk Score* dashboard.
 
 [role="screenshot"]
-image::images/hrs-all-hosts.png[]
+image::images/full-dashboard.png[Shows dashboard]
 
-The *Hosts by risk* tab on the Hosts page:
+Use the histogram to track how the risk score for a particular host has changed over time. To specify a date range, use the date and time picker, or drag and select a time range within the histogram.
+
+[role="screenshot"]
+image::images/histogram.png[]
+
+To go to the host's details page, left-click any host's corresponding bar in the histogram, then select *Go to Host View*.
 
 [role="screenshot"]
-image::images/hosts-by-risk-tab.png[]
+image::images/go-to-host-view.png[]
 
-The *Overview* section on the Host details page:
+The data tables beneath the histogram display associated rules, users, and MITRE ATT&CK tactics of risky hosts. The table data is sorted in reverse chronological order by default, with the highest total risk score at the top. Use this information to triage alerts with the highest risk in your network.
 
 [role="screenshot"]
-image::images/hrs-overview-section.png[]
+image::images/data-tables.png[]
+
+[[verify-host-risk-score]]
+=== Verify host risk score data installed successfully (Optional)
 
-The *Hosts by risk* tab on the Host details page:
+After enabling or upgrading host risk score, you may get the following message in the UI: 
 
 [role="screenshot"]
-image::images/hosts-by-risk-details-page.png[]
+image::images/restart-hrs.png[Restart host risk score]
+
+If so, it's recommended you verify that host risk score data has been generated by doing the following:
+
+In {kib}, run the following commands in Console to query the host risk score index: 
+
+[source,console]
+----------------------------------
+GET ml_host_risk_score_{{spaceID}}/_search
+GET ml_host_risk_score_latest_{{spaceID}}/_search
+----------------------------------
+
+If no data returns, you'll need to check if the source index (.`alerts-security.alerts-{{spaceID}}`) is generating alerts data:
+
+[source,console]
+----------------------------------
+GET _transform/<transform_id>/_stats?human=true
```

Comment:
Let's change this to be more straight forward:
GET _transform/ml_hostriskscore_pivot_transform_{{spaceID}}/_stats?human=true